src/components/transactions/Transactions.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="transactions">
+  <div id="home" class="transactions">
```

Comment:
is this ok? i think id=homas has been used in other divs. Ids should be unique, if you need a selector for css we should use a class instead

Reply:
fixed